### ISSUE TITLE
Vertebral artery tortuosity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7038]
**Preferred term label:**
Vertebral artery tortuosity

**Synonyms**
Tortuous vertebral arteries

**Definition (free text, please give PubMed ID)**
Abnormal tortuous (i.e., twisted) form of the carotid arteries.

**Parent term (use hpo.jax.org/app)**
Arterial tortuosity HP:0005116

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM:619329  4/4   PMID: 32938213

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also, celiac artery dissection
definition
A separation (dissection) of the layers of the celiac artery wall.  2/4 PMID: 32938213

**Comment by pnrobinson** [comment]
added
